middleware/index.ts

```diff
@@ -1,6 +1,8 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import {Logger, LoggerLevel} from '@salesforce/core/lib/logger';
+import {CloudEvent} from 'cloudevents-sdk/lib/cloudevent';
+import {HTTPReceiver} from 'cloudevents-sdk/lib/bindings/http/http_receiver';
```

Comment:
I'm a little mystified about why I have to import from the depths of the cloudevents-sdk library, but this seemed to make typescript (and the compiled javascript) happy.